lib/error.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var util = require('util');
+var ErrorCat = require('error-cat');
+var log = require('./logger');
+
+/**
+ * Custom ErrorCat error handler.
+ * @class
+ */
+function PonosError () {
+  ErrorCat.apply(this, arguments);
+}
+util.inherits(PonosError, ErrorCat);
+
+/**
+ * Logs errors via default ponos logger.
+ * @param {error} err PonosError to log.
+ * @see error-cat
+ */
+PonosError.prototype.log = function (err) {
```

Comment:
So the way we handle error logging in our internal projects is to use [error-cat](https://github.com/runnable/error-cat). Should we eat our own cat food? We could technically have ponos use error-cat and monitor-dog by default. And give the user a way of redefining the instances (in the case they have done custom implementations in their projects).